packages/apps/patterns/react-composer/src/testing/replicator.ts

```diff
@@ -175,11 +179,12 @@ export class Replicator {
 
 export type UseYjsModelOptions = {
   replicator: Replicator;
+  id: string;
   doc?: Doc;
 };
 
-export const usePlainYjsModel = ({ replicator, doc }: UseYjsModelOptions): PlainTextModel => {
-  const peer = useMemo(() => replicator.createPeer(doc), [doc]);
+export const useYjsModel = ({ replicator, id, doc }: UseYjsModelOptions): ComposerModel => {
```

Comment:
Can we promote a more general useTextModel to react-client that can be parameterized by text type?